errors/no-head-import-in-document.md

```diff
@@ -1,5 +1,11 @@
 # No Head Import in Document
 
+> Prevent usage of `next/head` in `pages/_document.js`.
+
+### Why This Error Occurred
+
+`next/head` should not be imported in `pages/_document.js`. This can cause unexpected issues in your application.
+
 ### Why This Error Occurred
 
 `next/head` was imported in `pages/_document.js`. This can cause unexpected issues in your application.
```

Comment:
I think "was imported" is more clear for _why_ it occurred.

Reply:
Fixed in https://github.com/vercel/next.js/pull/34335/commits/d16be8bfdce8c73c0c4eb80d48cdfb85de6b8bae.